tasks/eth/ink-001-permissionless-proofs/.env

```diff
@@ -0,0 +1,9 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+SAFE_NONCE=""
```

Comment:
Might be worth specifying a nonce since there will be other tasks that succeed this. This avoids accidental out of order execution.

Reply:
Added the nonce of the 2/2